code/core/src/csf/story.ts

```diff
@@ -165,10 +168,10 @@ export interface Globals {
   [name: string]: any;
 }
 export interface GlobalTypes {
-  [name: string]: InputType;
+  [name: string]: ToolbarArgType;
 }
 export interface StrictGlobalTypes {
-  [name: string]: StrictInputType;
+  [name: string]: StrictInputType & ToolbarArgType;
```

Comment:
I'm not particularly happy about this one either to be honest. Should `GlobalTypes` even be using `InputType` as its bas? How much of InputType does it actually use? `defaultValue` is marked as deprecated in `InputType` but _is_ used in `ToolbarArgType`. Perhaps we should decouple these types?